### ISSUE TITLE
Bug : MultiLevel category names fail for non ASCII names

### ISSUE THREAD

**Issue by biggihs** [scanny/python-pptx#276]
The method `_lvl_xml` fails when the `name` variable has non-ascii values because it uses the `str` method on the `name`. Changing it to `unicode` fixes the issue.

File : `/pptx/chart/xmlwriter.py`
``` 
1524     def _lvl_xml(self, categories):
1525         """
1526         The unicode XML snippet for the ``<c:lvl>`` elements containing
1527         multi-level category names.
1528         """
1529         def lvl_pt_xml(level):
1530             xml = ''
1531             for idx, name in level:
1532                 xml += (
1533                     '                  <c:pt idx="%d">\n'
1534                     '                    <c:v>%s</c:v>\n'
1535                     '                  </c:pt>\n'
1536                 ) % (idx, escape(str(name)))
1537             return xml```

**Comment by scanny** [comment]
Ah, okay, I should have read further down in my email, I see now. Thanks, will get this in with upcoming release.

**Comment by scanny** [comment]
This is fixed in release v0.6.6.
